package.json

```diff
@@ -1,45 +1,48 @@
 {
-  "name": "@shapirov/cdk-eks-blueprint",
-  "version": "0.1.5",
-  "main": "lib/index.js",
-  "types": "lib/index.d.ts",
-  "bin": {
-    "main": "bin/main.js"
-  },
-  "scripts": {
-    "build": "tsc",
-    "watch": "tsc -w",
-    "test": "jest",
-    "cdk": "cdk"
-  },
-  "devDependencies": {
-    "@aws-cdk/assert": "1.91.0",
-    "@types/jest": "^26.0.15",
-    "@types/js-yaml": "^3.12.5",
-    "@types/node": "10.17.27",
-    "aws-cdk": "1.91.0",
-    "jest": "^26.6.1",
-    "ts-jest": "^26.4.3",
-    "ts-node": "^8.1.0",
-    "typescript": "~3.9.7"
-  },
-  "dependencies": {
-    "@aws-cdk/aws-autoscaling": "1.91.0",
-    "@aws-cdk/aws-codebuild": "1.91.0",
-    "@aws-cdk/aws-codecommit": "1.91.0",
-    "@aws-cdk/aws-codepipeline": "1.91.0",
-    "@aws-cdk/aws-codepipeline-actions": "1.91.0",
-    "@aws-cdk/aws-dynamodb": "1.91.0",
-    "@aws-cdk/aws-ec2": "1.91.0",
-    "@aws-cdk/aws-ecr": "1.91.0",
-    "@aws-cdk/aws-eks": "1.91.0",
-    "@aws-cdk/aws-elasticsearch": "1.91.0",
-    "@aws-cdk/aws-iam": "1.91.0",
-    "@aws-cdk/aws-logs": "1.91.0",
-    "@aws-cdk/aws-secretsmanager": "1.91.0",
-    "@aws-cdk/core": "1.91.0",
-    "@aws-cdk/pipelines": "1.91.0",
-    "js-yaml": "3.13.1",
-    "sync-request": "6.1.0"
-  }
-}
+    "name": "@shapirov/cdk-eks-blueprint",
+    "version": "0.1.5",
+    "main": "dist/lib/index.js",
+    "types": "dist/lib/index.d.ts",
+    "bin": {
+        "main": "dist/bin/main.js"
+    },
+    "scripts": {
+        "build": "rm -rf dist && tsc",
```

Comment:
We really need a `clean` target here to remove previously generated files like .js and .d.ts since those caused compile issues. tsc could not find them since you removed the ts file (renamed to index).